CIP-0105/README.md

```diff
@@ -102,37 +103,37 @@ These are also described in [CIP-0005 | Common Bech32 Prefixes](https://github.c
 
 DRep keys and DRep IDs should be encoded in Bech32 with the following prefixes:
 
-| Prefix     | Meaning                                   | Contents                           |
-| ---------- | ----------------------------------------- | ---------------------------------- |
-| `drep_sk`  | CIP-1852’s DRep signing key               | Ed25519 private key                |
-| `drep_vk`  | CIP-1852’s DRep verification key          | Ed25519 public key                 |
-| `drep_xsk` | CIP-1852’s DRep extended signing key      | Ed25519-bip32 extended private key |
-| `drep_xvk` | CIP-1852’s DRep extended verification key | Ed25519 public key with chain code |
-| `drep`     | DRep credential                           | DRep credential                    |
+| Prefix     | Meaning                                                 | Contents                                                          |
+| ---------- | --------------------------------------------------------| ----------------------------------------------------------------- |
+| `drep_sk`  | CIP-1852’s DRep signing key                             | Ed25519 private key                                               |
+| `drep_vk`  | CIP-1852’s DRep verification key                        | Ed25519 public key                                                |
+| `drep_xsk` | CIP-1852’s DRep extended signing key                    | Ed25519-bip32 extended private key                                |
+| `drep_xvk` | CIP-1852’s DRep extended verification key               | Ed25519 public key with chain code                                |
+| `drep`     | Delegate representative verification key hash (DRep ID) | blake2b\_224 digest of a delegate representative verification key |
```

Comment:
the `drep` prefix is included in CIP-0105, so i thought if `drep` is included, why not also mention `drep_script` here to give a full picture?